src/schema/v1/partner_show.ts

```diff
@@ -363,16 +362,20 @@ const PartnerShow: GraphQLFieldConfig<void, ResolverContext> = {
   type: PartnerShowType,
   description: "A Partner Show",
   args: {
-    id: {
+    partner_id: {
+      type: new GraphQLNonNull(GraphQLString),
+      description: "The slug or ID of the Partner",
+    },
+    show_id: {
       type: new GraphQLNonNull(GraphQLString),
       description: "The slug or ID of the PartnerShow",
     },
   },
-  resolve: (_root, { id }, { showLoader }) => {
-    return showLoader(id).then(show => {
-      if (!show.displayable) {
-        return new HTTPError("Show Not Found", 404)
-      }
+  resolve: (_root, { partner_id, show_id }, { partnerShowLoader }) => {
+    return partnerShowLoader({
+      partner_id: partner_id,
+      show_id: show_id,
+    }).then(show => {
```

Comment:
As mentioned above, the `PartnerShow` type is deprecated so these changes should go to the `Show` type instead.